static/js/containers/pages/CheckoutPage.js

```diff
@@ -174,7 +174,7 @@ export class CheckoutPage extends React.Component<Props, State> {
       coupons: couponCode
         ? [
           {
-            code: couponCode
+            code: couponCode.trim()
```

Comment:
@tasawernawaz Can we support this change with a test ?

Reply:
added